plugins/modules/ec2_vol.py

```diff
@@ -439,6 +439,22 @@ def update_volume(module, ec2_conn, volume):
             volume["multi_attach_enabled"] = response.get("VolumeModification").get("TargetMultiAttachEnabled")
             volume["throughput"] = response.get("VolumeModification").get("TargetThroughput")
 
+            if module.params.get("wait_for_modify_volume_complete"):
+                mod_state = ""
+                _max_attempts = 100
+                _sleep_secs = 10
+                _attempt = 0
+                while mod_state != "completed":
```

Comment:
Would it be worthing adding a check for `failed` state here too and exiting with an error in that case?

Reply:
Hi. I will add checking for `failed`. Great idea. What value should I put for `version_added` in the description?

Reply:
I think 6.3.0 is our next release.

Reply:
ok I made those changes. I also increased the polling interval as, even for small volumes, it can take a really long time to finish